pkg/e2e/infra/utils.go

```diff
@@ -437,14 +437,11 @@ func waitUntilAllRCPodsAreReady(client runtimeclient.Client, rc *corev1.Replicat
 			return false, nil
 		}
 		if rcObj.Status.ReadyReplicas == 0 {
-			glog.Infof("Waiting for at least one RC ready replica (%v/%v)", rcObj.Status.ReadyReplicas, rcObj.Status.Replicas)
+			glog.Infof("Waiting for at least one RC ready replica, ReadyReplicas: %v, Replicas: %v", rcObj.Status.ReadyReplicas, rcObj.Status.Replicas)
```

Comment:
Do the same change in a few lines below for consistency now.

Reply:
Done